src/content/PostsContent.tsx

```diff
@@ -94,16 +94,16 @@ const posts: PostDescription[] = [
     isInModelGenerationAppPage: true,
   },
   {
-    title: 'Using BPMN Visualization in R',
-    text: 'The new BPMN Visualization - R Package is being developed to offer another way to visualize process execution data in R.',
+    title: 'Using bpmn-visualization in R',
```

Comment:
We shouldn't update past articles afterwards. This has never been the purpose of the renaming in my mind.
In the code, we don't update past commits nor past documentation. So we shouldn't do it for articles as well.

I guess there is a misunderstanding here.

This comment applies to all related changes currently proposed in this file.